compiler+runtime/src/cpp/jank/runtime/module/loader.cpp

```diff
@@ -561,7 +567,7 @@ namespace jank::runtime::module
         return err(
           fmt::format("unable to map file {} due to error: {}", entry.path, file.expect_err()));
       }
-      rt_ctx.jit_prc.eval_string({ file.expect_ok().head, file.expect_ok().size });
+      rt_ctx.eval_cpp_string({ file.expect_ok().head, file.expect_ok().size });
```

Comment:
When we were talking in Slack, we noted how the `load_cpp` fn needs to call the `jank_load_` fn. Let's include that in this PR, please, since `.cpp` modules won't work properly without it (until they're AOT compiled).

Reply:
`extern "C"`s in the cpp module open the same hole that we covered by introducing the symbol counters in jank sources.

Suppose user has two modules `foo.cpp` and `bar.cpp` and they both define a function `hello_world`

```cpp
// foo.cpp
extern "C" void hello_world() {
    cout<<"Hello from foo\n";
}
```

```cpp
// bar.cpp
extern "C" void hello_world() {
    cout<<"Hello from bar\n";
}
```

Trying to load both modules causes error because of duplicate definition of the `hello_world` function. Reloading a, already loaded C++ module, causes the same error.

Not sure how big of a problem it is. People may have conflicting extern defines that conflict with a library defines?
This PR leaves that problem open as of now.

Reply:
Yep, this would be a problem with *any* C++ program, so it's not going to be a surprise to any C++ dev.